doc/en/user/source/datadirectory/migrating.rst

```diff
@@ -153,4 +153,33 @@ Backup files
                config.xml.2.2.x
           config.xml.2.2.x        
 
-       
\ No newline at end of file
+Migrating between GeoServer 2.5.x and 2.7.x
```

Comment:
I think we should backport this change, and change this to GeoServer 2.5.x and GeoServer 2.6.x